### ISSUE TITLE
0.40.0: JUnit is in the compile scope

### ISSUE THREAD

**Issue by markkolich** [vsch/flexmark-java#292]
Hi @vsch, I was tempted to re-open https://github.com/vsch/flexmark-java/issues/291 but I opted to file a separate bug instead.

I migrated my project to `com.vladsch.flexmark:flexmark` version `0.40.0` and I still see `junit:junit` in the `compile` scope:

```
[INFO] +- com.vladsch.flexmark:flexmark:jar:0.40.0:compile
[INFO] |  +- com.vladsch.flexmark:flexmark-test-util:jar:0.40.0:compile
[INFO] |  |  \- junit:junit:jar:4.12:compile << -- !!
[INFO] |  |     \- org.hamcrest:hamcrest-core:jar:1.3:compile
[INFO] |  \- com.vladsch.flexmark:flexmark-util:jar:0.40.0:compile
```

Is that intentional?

**Comment by vsch** [comment]
@markkolich, I will check, I thought I marked the dependency as test scope.

**Comment by vsch** [comment]
@markkolich, I removed the code but left in the redundant import and forgot to add test scope in the pom.

0.40.2 fixes this.

**Comment by vsch** [comment]
Fix for this is available. Repo updated, maven updated but may take a while to show up in maven central.

**Comment by markkolich** [comment]
@vsch thanks, will try with `0.40.2` later today!

**Comment by markkolich** [comment]
@vsch confirmed fixed with `0.40.2`. Thanks!

Closing.
